### DISCUSSION TITLE
JWT authentication, dealing with OPTIONS in the preflight

### DISCUSSION THREAD

**Discussion by edsonribeiro13** [prest/prest#810] [API Server] [answered]
As I have reported (https://github.com/prest/prest/issues/802) apparently there's a unexpected way of dealing with OPTIONS requests in the preflight when JWT is enabled. So we have two solutions available: send authorization as a param in the URL (example on the original Issue) and/or have a pre-response ready for OPTIONS methods.

**Comment by avelino** [comment]
@Listener430 is working on solving issue 802

PR: https://github.com/prest/prest/pull/811

**Comment by arxdsilva** [comment, marked as answer]
@edsonribeiro13 we've just released version `v1.2.6` that should fix this issue, could you have a look?

**Reply by edsonribeiro13** [reply]
Worked like a charm, thanks!
